src/Reports/Reports.tsx

```diff
@@ -1,11 +1,89 @@
-import React, { Component } from "react";
+import React from "react";
+import { BarChart, Bar, CartesianGrid, XAxis, YAxis, Label, Cell, Line, ComposedChart } from 'recharts';
+import { JournalEntry } from "../journal";
+import { JournalContext, useJournal } from "../JournalContext";
+import { scaleLinear } from 'd3-scale';
```

Comment:
this is not an issue but something that I try to do. when adding imports to a file I try to separate out dependencies (eg react) from the code I write so the above would like something like this:

```js
import React from "react";
import { BarChart, Bar, CartesianGrid, XAxis, YAxis, Label, Cell, Line, ComposedChart } from "recharts";
import { scaleLinear } from "d3-scale";

import { JournalEntry } from "../journal";
import { JournalContext, useJournal } from "../JournalContext";
```